src/Bicep.Core/Configuration/ExperimentalFeaturesEnabled.cs

```diff
@@ -14,7 +14,8 @@ public record ExperimentalFeaturesEnabled(
     bool? SourceMapping,
     bool? ParamsFiles,
     bool? UserDefinedTypes,
-    bool? UserDefinedFunctions)
+    bool? UserDefinedFunctions,
+    bool? DynamicTypeLoading)
```

Comment:
Will need to update https://github.com/Azure/bicep/blob/main/src/vscode-bicep/schemas/bicepconfig.schema.json